### DISCUSSION TITLE
✨[Feature] New Release for pip

### DISCUSSION THREAD

**Discussion by dignakov** [pytorch/TensorRT#913] [Q&A]
Would it be possible to get a new release for use with pip?

There have been quite a few features and bug-fixes added since November, and it would be great to have an up to date version available.

I know that docker containers are often recommended, but that's often not a viable option.

Thank you for all of the great work!!

**Comment by narendasan** [comment]
A new release will be coming with the new version of PyTorch (1.11.0). Usually you can expect a new release from us around the time they release a new version.
